backend/.rgignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+dist
```

Comment:
This file is for ripgrep, or something else?

Reply:
It is. My mistake.